os_migrate/roles/import_workloads/tasks/main.yml

```diff
@@ -57,6 +66,9 @@
     src_auth_type: "{{ os_migrate_src_auth_type|default(omit) }}"
     src_region_name: "{{ os_migrate_src_region_name|default(omit) }}"
     src_validate_certs: "{{ os_migrate_src_validate_certs|default(omit) }}"
+    src_ca_cert: "{{ os_migrate_src_ca_cert|default(omit) }}"
```

Comment:
I think we need to comment out these few lines and mark them "TODO" as the module currently won't accept these arguments, so running this role with them present would probably crash. Let's comment these out for now and create a tech debt issue?